frontend/src/components/Table/PropertyTypeCell.tsx

```diff
@@ -5,7 +5,7 @@ import { ReactComponent as LandSvg } from 'assets/images/icon-lot.svg';
 import { ReactComponent as SubdivisionSvg } from 'assets/images/project-diagram-solid.svg';
 
 import { IProperty } from 'actions/parcelsActions';
-import { PropertyTypes } from 'actions/parcelsActions';
+import { PropertyTypes } from 'constants/propertyTypes';
```

Comment:
for anyone else who is running into this, ideally what we should be doing is:
import { PropertyTypes, ... } from 'constants';
however, the above is failing because it can't find the index within the constants folder. Not sure why yet.
In the meantime you can work around this by using 
import { PropertyTypes, ... } from 'constants/index';

Reply:
not necessary here because we only need the one constant.

Reply:
I believe this is because one of our npm packages has a `constants`.